lb_content_resolver/model/directory.py

```diff
@@ -0,0 +1,18 @@
+from peewee import *
+from lb_content_resolver.model.database import db
+
+
+class Directory(Model):
```

Comment:
I wonder if DirectoryCache or DirectoryMTime would be a better name. Directory is too generic in this case, I think.